### PR TITLE
feat(aws): Add New Elasticache Resources and Attributes

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: plugins/source/aws/docs/tables/aws_elasticache_events.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|region|String|
+|date (PK)|Timestamp|
```
Comment (erezrokah) [inline plugins/source/aws/docs/tables/aws_elasticache_events.md:17]: Does this have enough resolution to be unique? Can two events happen at the same time?

**Review by erezrokah — CHANGES_REQUESTED** [review]
Looks great 🚀  One comment on the PK

File: plugins/source/aws/docs/tables/aws_elasticache_events.md (hunk 1 above)
Comment (bbernays) [reply plugins/source/aws/docs/tables/aws_elasticache_events.md:17]: I don't know... All of the other fields can be empty... So not great candidates for PK... Maybe Hash the object unless you have another idea

Comment (hermanschaaf) [reply plugins/source/aws/docs/tables/aws_elasticache_events.md:17]: Yeah let's rather hash the object if we're not certain 👍

**Review by erezrokah — APPROVED** [review]

**Pull request merged**